packages/ignition/src/app/locales/docs/usage/shortcuts.en.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Shortcuts
+description: Shortcuts usage
+slug: shortcuts
+order: 9
+---
+
+import RocketIconsText, { RocketIconsTextDefault } from "@/components/rocketicons-text";
+
+## Usage
+
+# What is shortcuts?
+
+Using <RocketIconsText /> to style icons on dark mode.
+
+You can style icons differently when dark mode is enabled.
```

Comment:
Is this text correct for this page? The pt-br one seems to be OK.

Reply:
Fixed